tests/test_computers.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""Test the /computers endpoint"""
+
+
+def test_get_computers(default_computers, client):  # pylint: disable=unused-argument
+    """Test listing existing computer."""
+    response = client.get("/computers/")
+
+    assert response.status_code == 200
+    assert len(response.json()) == 2
+
+
+def test_get_computers_projectable(client):
+    """Test get projectable properites for computer."""
+    response = client.get("/computers/projectable_properties")
+
+    assert response.status_code == 200
+    assert response.json() == [
+        "id",
+        "uuid",
+        "name",
+        "hostname",
+        "scheduler_type",
+        "transport_type",
+        "metadata",
+    ]
+
+
+def test_get_single_computers(
+    default_computers, client, authenticate
+):  # pylint: disable=unused-argument
+    """Test retrieving a single computer."""
+    response_post = client.post(
```

Comment:
This works.
To make the test more specific, as mentioned the `default_computers` fixture could simply return the ids of the computers it created, which you could then reuse here.